Spectre (Update)

ThatOneGoblin25 opened the pull request:
## About the PR
- Reworked all rooms
- Wiring rework
- Atmos rework
- Decal rework
- Added atmos piping to artifact chambers
- Added robotics
- Added plasma, carbon and empty canisters
- Added a power drill as a RD item
- Added a science techfab
- Added a television to bar
- Added Donut Vend
- Added a Genedrobe with a secret wall below it
- Replaced most consoles with tabletop variants
- Removed blast doors over artifact chamber entrance
- Removed medical doctor's wall locker
- Removed some of the artifact containers
- Trimmed the amount of lights
- Tweaked bar, booze-o-mat instead of soda dispenser
- Tweaked price, from 195000 to 1850000
- Renamed from NSS Spectre to NR Spectre
- New description.

## Why / Balance
Updating to new mapping standards, also only large ship to be missing a mothership console.

## Media
![Screenshot 2024-03-23 003631](https://github.com/new-frontiers-14/frontier-station-14/assets/145570657/b19460c0-6dd0-44af-8451-e4c94034f7f9)
![Screenshot 2024-03-23 003618](https://github.com/new-frontiers-14/frontier-station-14/assets/145570657/abd00c5f-af09-45af-b59e-7929b7fe98f8)
![Screenshot 2024-03-23 003554](https://github.com/new-frontiers-14/frontier-station-14/assets/145570657/b591ce58-9dbe-41db-a32d-1f22c74b02e5)
![Screenshot 2024-03-23 003533](https://github.com/new-frontiers-14/frontier-station-14/assets/145570657/932e3673-c2a3-4a1e-81ec-d43cfc350794)


- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- Tweak: Updated the Spectre to current mapping guidelines with additional features.

ThatOneGoblin25 commented:
@Wolfhauler Letting you know I've proposed some changes to your ship.

ThatOneGoblin25 commented:
Map checker failed because of the telecomms having Cargo, Engineering, Science and Service encryption keys.

Wolfhauler commented:
o7

Wolfhauler commented:
crematorium was denied last time by the by

MagnusCrowe commented:
I don't understand why a research ship would have a mothership console that allows you to get Pioneer, Prospector, Chisel, Bocadillo, PTS, Construct.  It makes no sense to me.

Why the Smile AI?

ThatOneGoblin25 commented:
> I don't understand why a research ship would have a mothership console that allows you to get Pioneer, Prospector, Chisel, Bocadillo, PTS, Construct. It makes no sense to me.
Drawing from its description: "A large research mothership designed to be flown next to a small fleet of other ships including salvage and food services." It should be able to provide these ships conveniently to the crew on board. The Crescent mothership console provides all the ships I listed and more. 
There wasn't much point I saw in adding additional sci ships like the Sparrow or Investigator, since they do the same job as the main ship and we want to keep activity on there. I did have a vague idea to make a unique ship, a solar-powered shuttle that only had an artifact scanner and analysis console for this console but I realised it would detract from what the Spectre could offer.

As for the Smile AI, it's something unique that draws parallels to sci labs on stations. The largest sci-ship on the market should draw similarities to such sci labs. But it can go as the ghost role isn't whitelisted, you can't DO anything other than talk or suicide but if you believe it unnecessary I can remove it.

MagnusCrowe commented:
Nah, keep Smile.  Makes more sense for it to be on the Spectre than the Wasp anyway

dvir001 commented:
> Nah, keep Smile. Makes more sense for it to be on the Spectre than the Wasp anyway

Fucker worth 3k dead

MagnusCrowe commented:
> > Nah, keep Smile. Makes more sense for it to be on the Spectre than the Wasp anyway
> 
> Fucker worth 3k dead

I feel like that is a separate issue that should be addressed...why is it worth that much?

ThatOneGoblin25 commented:
I really feel like the Spectre deserves a mothership console still. Prehaps nerfed slightly in its options. It's really a sci base away from station and a substitute for Anom Labs with the Anomaly generator. As for docks, even with only two I think it can be managed by any crew, it is a tiny obstacle managing the docks.

Also map checker fails for the encryption keys in the telecomms machine, it's something the original creator added, should I remove it @dvir001 ?

ThatOneGoblin25 commented:
Ignore the commit spaghetti
Updated PR screenshots
- Removed the mothership console and textures
- Added a donut vendor, for a large crew have *no* food feels a bit unfair, especially since food services aren't guaranteed. Half the donuts are poisonous to felinids and vulps which is hilarious.
- Added a borg recharger
- Made gyros look better. This requires LV cables to run under reinforced walls but its rather minor, and is only used to power two gyros on the tips of the ship. I felt this was necessary to preserve the sleek ship design with minimal sacrifices.

Cheackraze reviewed (approved):
Im not personally opposed to the _idea_ of having the mothership console here, the problem I have is that it should be a very narrow, restricted 'themed' list of mini science drone ships, instead of just a generic list of food/general RP ships. The rest of the changes here are fine

dvir001 commented:
No food vendors on ships, having the restock option make it infinite food machine that remove the need to leave it.

dvir001 commented on Resources/Prototypes/_NF/Shipyard/spectre.yml:
```diff
@@ -1,15 +1,25 @@
 - type: gameMap
   id: Spectre
-  mapName: 'NSSSpectre'
+  mapName: 'Spectre'
```
Please use the same map name format as all other ships

ThatOneGoblin25 replied:
Done

ThatOneGoblin25 commented:
Removed Donut Vendor
Removed Drill
Removed Two Canisters
Removed slips

ThatOneGoblin25 commented:
> the problem I have is that it should be a very narrow, restricted 'themed' list of mini science drone ships

I am willing to do this. Please hold off on merging.

dvir001 commented on Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/mothership-computers.yml:
```diff
@@ -100,6 +100,27 @@
+# The spectre console
+- type: entity
+  id: SpectreMothershipComputer
+  name: Spectre mothership console
```
Lower case in names

dvir001 commented on Resources/Prototypes/_NF/Shipyard/ghost.yml:
```diff
@@ -0,0 +1,36 @@
+# Shuttle Notes:
+#
+- type: vessel
+  id: PTS
```
You cannot just copy paste an existing entity

MagnusCrowe commented:
I cannot actually build the code in this PR so I can't review it at this time.  Does it build for you?

ThatOneGoblin25 commented:
Added the NX Ghost shuttle for the Spectre Mothership Console.
![Screenshot 2024-04-02 161034](https://github.com/new-frontiers-14/frontier-station-14/assets/145570657/13d1cdf4-7b3b-4af2-ae16-e4f6d8216416)

ThatOneGoblin25 commented:
I am like 90% sure i screwed up my own fork and if I reopened this on a new branch this wouldn't happen :P

ThatOneGoblin25 commented:
Let's try opening a new branch.
